### PR TITLE
vu ami update

### PR DESCRIPTION
**Pull request by pbruski**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/AmiNameResolver.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.awsinfrastructure.api.virtualusers
+
+import com.amazonaws.services.ec2.model.DescribeImagesRequest
+import com.amazonaws.services.ec2.model.Filter
+import com.atlassian.performance.tools.aws.api.Aws
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentMap
+
+internal class AmiNameResolver {
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/AmiNameResolver.kt]: `internal` shouldn't be in `api`

File: src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/StackVirtualUsersFormula.kt [hunk 2]
```diff
@@ -89,9 +89,11 @@ class StackVirtualUsersFormula private constructor(
     ): ProvisionedVirtualUsers<SshVirtualUsers> {
         logger.debug("Setting up $name...")
         val network = overriddenNetwork ?: NetworkFormula(investment, aws).provision()
+        val cloudformationTemplate = readResourceText("aws/virtual-users.yaml")
+        logger.info("template: $cloudformationTemplate")
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/StackVirtualUsersFormula.kt]: What is it for? Isn't it the same as reading from source code directly?

File: src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/AmiNameResolver.kt (hunk 1 above)
Comment (pbruski) [reply src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/AmiNameResolver.kt]: moved out

File: src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/StackVirtualUsersFormula.kt (hunk 2 above)
Comment (pbruski) [reply src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/StackVirtualUsersFormula.kt]: debug, removing

**Pull request merged**